### PR TITLE
updating docs example multiple shoot

### PR DESCRIPTION
**Pull request by yewalenikhil65**
1. included plots call for single-prediction from multiple -shooting.. in commented form!
2. `solver` as choice of ODEsolver included

[c961c16](https://github.com/SciML/DiffEqFlux.jl/pull/544/commits/c961c167e9b643d66a315d21d567b9c6e6f2e859) is resolved already sometime back.. Yet i don't know why its appearing in this PR also

### CODE REVIEW

File: docs/src/examples/multiple_shooting.md [hunk 1]
```diff
@@ -33,8 +33,9 @@ function trueODEfunc(du, u, p, t)
     true_A = [-0.1 2.0; -2.0 -0.1]
     du .= ((u.^3)'true_A)'
 end
+solver =Tsit5()
 prob_trueode = ODEProblem(trueODEfunc, u0, tspan)
-ode_data = Array(solve(prob_trueode, Tsit5(), saveat = tsteps))
+ode_data = Array(solve(prob_trueode, solver, saveat = tsteps))
```
Comment (Vaibhavdixit02) [inline docs/src/examples/multiple_shooting.md:38]: Can you say more about why you think this helps?

File: docs/src/examples/multiple_shooting.md [hunk 2]
```diff
@@ -66,7 +67,12 @@ callback = function (p, l, preds; doplot = true)
 
 	# plot the different predictions for individual shoot
 	plot_multiple_shoot(plt, preds, group_size)
-
+        
+	# plot the single prediction using multiple shoot
+	#prediction  = solve(remake(prob_node, p=p, tspan=(tsteps[1],tsteps[datasize]),u0=ode_data[:,1]), solver, saveat=tsteps)
+	#plot!(plt, tsteps, prediction[1,:],label = "Prediction")
```
Comment (Vaibhavdixit02) [inline docs/src/examples/multiple_shooting.md:73]: I don't see much point in adding commented out code, why not actually run and show the plots with some description?

**Comment by ChrisRackauckas** [comment]
yeah I agree, I don't get this PR.

**Comment by yewalenikhil65** [comment]
> why not actually run and show the plots with some description?

@Vaibhavdixit02 I can do this ,if it's good to have in example. Sorry for replying late as I am isolated. 

>yeah I agree, I don't get this PR.

@ChrisRackauckas Hi sir, I agree there isn't much here, but thought to include single prediction plot alongside group predictions might be good(I think in earlier version it was there). If it's not needed, we can close this PR.

**Comment by Vaibhavdixit02** [comment]
Add the plot and some description why you think it's useful then the PR would be useful 🙂

**Pull request closed without merging**